### ISSUE TITLE
Consider city and country in geocoding requests

### ISSUE THREAD

**Issue by littlepea** [littlepea/safeway-data#10]
This is an enhancement on top of #5 based on [this discussion](https://github.com/littlepea/safeway-data/pull/7#discussion_r878217908).

We could:

1. pass city and country as explicit parameters to the geocoding request if the API supports it
2. append the city and country to the address separated by commas
3. use country-specific address parsing library (probably overkill)

**Comment by wdunn001** [comment]
I thought about what we should do for the address issue. I think we should utilize google places for this problem. Before we geocode we should first do an address validation process using https://developers.google.com/maps/documentation/javascript/places-autocomplete
Google Developers
Places Autocomplete  |  Maps JavaScript API  |  Google Developers
Places Autocomplete  |  Maps JavaScript API  |  Google Developers
and just take the first option as it will be the most likely match

**Comment by wdunn001** [comment]
@littlepea I am making an address sanitizer. What would we use to signify the address has been sanitized already?

**Comment by littlepea** [comment]
> @littlepea I am making an address sanitizer. What would we use to signify the address has been sanitized already?

It would be based on the source, let’s just add a flag to the ‘main.py’ to opt into address sanitization (add it in settings and let `main.py` to override it)

**Comment by littlepea** [comment]
I'm not sure that's the value of that, the most important is to get coordinates

**Comment by littlepea** [comment]
@wdunn001 did you actually test this? I was getting `TypeError` when trying to run it... (fixed in #37)

And most requests are failing:

```
Loading records from file data/mapahelp.csv
Failed to sanitize address: ul. Oleandrów, 6 lok. 2, 00-629 Warszawa, 1 piętro. REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: . INVALID_REQUEST
24 Arsena Street has been converted to 24 Arsena Street, Tbilisi, Georgia
Проспект Чавчавадзе 5 / 14 has been converted to 5 Ilia Chavchavadze Avenue, Tbilisi, Georgia
34 Lado Asatiani St, Tbilisi has been converted to 34 Lado Asatiani St, Tbilisi, Georgia
Failed to sanitize address: ул. Griashashvili 56. 
Failed to sanitize address: Станція метро «Теремки». REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: Станція метро «Бориспільська». REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: Станція метро «Вирлиця». REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: Станція метро «Осокорки». REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: Станція метро «Позняки». REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: Станція метро «Славутич». REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: Станція метро «Харківська». REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: Станція метро «Червоний Хутір». REQUEST_DENIED (This API project is not authorized to use this API.)
Failed to sanitize address: м. Київ, просп. Броварський (ст. м. "Чернігівська"). REQUEST_DENIED (This API project is not authorized to use this API.)
Станція метро «Гідропарк» has been converted to Гідропарк, Kyiv, Ukraine
Станція метро «Дарниця» has been converted to Дарниця, Kyiv, Ukraine
Failed to sanitize address: Станція метро «Героїв Дніпра». REQUEST_DENIED (This API project is not authorized to use this API.)
```
